lib/tasks/create_mbrave_tags.rake

```diff
@@ -0,0 +1,205 @@
+# frozen_string_literal: true
+require 'csv'
+require 'yaml'
+
+namespace :mbrave do
+  desc 'Modifying tag groups'
+
+  #
+  # How to use it:
+  # bundle exec rake mbrave:create_tag_plates <tag_layout_template> <login> <num_plates>
+  #
+  # Example:
+  # bundle exec rake mbrave:create_tag_plates Bioscan_384_template_1_v3 admin 5
+  task :create_tag_plates, %i[arg1 arg2] => :environment do |_t, _args|
```

Comment:
I'm not sure I follow the syntax here as I've not seen it before, but I think this is declaring that you should pass in two arguments, but then they're not used in this task because you've prefixed `_t` and `_args` with underscores.  Are they used by the `:environment` task?  It would probably be good to name these arguments if they're needed, even if it's not here.

Or is this just a way to ensure the user passes in arguments?  You seem to check for `ARGV` in your task which I think is a (wrong?) way to access the arguments for the task.  Perhaps there is something that can be done here to make things clearer.  Like access the args via the `_args` parameter and naming them appropriately.